modules/rpc/thriftrw-plugin-thriftsync/README.md

```diff
@@ -3,17 +3,44 @@
  *.thrift file. With the use of thriftsync plugin, a user who needs to build a service should be able
  to auto generate the code and write service specific logic without worrying about underlying platform.
 
+## Usage
+Run `thriftsync` on any thrift file update to sync your handler code with the methods in the thrift file.
+
+*Install thriftsync from vendor:*
+`go install ./vendor/go.uber.org/fx/modules/rpc/thriftrw-plugin-thriftsync`
+
+*Run thriftrw code genration with thriftsync*
+`thriftrw --plugin="thriftsync --yarpc-server=<Import path of the yarpc servicenameserver>" <thrift filepath>`
+
+Update your makefile with following lines and simply run `make thriftsync`
+*Update makefile*
```

Comment:
The steps in italics don't render well. Are you OK with numbering them instead, with step 1 in line 9?